model:
  # ['sparse_VGG', 'dense_VGG', 'sparse_RNN', 'dense_RNN']
  model_name: 'sparse_RNN' #'fb_sparse_object_det'
  yolo_threshold: 0.6
  yolo_iou: 0.1
optim:
  batch_size: 1
  init_lr: 1e-4
  # Set to "steps: ~" if no scheduler is needed
  steps_lr: [500, 1000, 1500]
  factor_lr: 0.1
checkpoint:
  resume_training: False
  resume_file: ''
  # Networks trained for classification can be reused as starting point for object detection networks on trained
  # the same dataset
  use_pretrained: True

  pretrained_dense_vgg: '../../../trained_models/dense_VGG.pth'
  pretrained_sparse_vgg: '../../../trained_models/sparse_VGG.pth'
  pretrained_sparse_rnn: '../../../trained_models/sparse_RNN.pth'
hardware:
  # num_cpu_workers: {-1: auto, 0: main thread, >0: ...}
  num_cpu_workers: 0
  gpu_device: 'cpu'  # [0 or 'cpu']
